internal/provider/resource_consul_cluster.go

```diff
@@ -534,6 +565,19 @@ func setConsulClusterResourceData(d *schema.ResourceData, cluster *consulmodels.
 		}
 	}
 
+	var ipAllowlist []interface{}
```

Comment:
This part in `func setConsulClusterResourceData(...)` is used for the read operation. See this [line](https://github.com/hashicorp/terraform-provider-hcp/blob/99e5cb783f73cabfcbfa167c8d32612b56e67150/internal/provider/resource_consul_cluster.go#L587). Nothing else is needed for the read.